child-process.ts

```diff
@@ -79,11 +79,17 @@ export class ChildProcess implements IChildProcess {
 					if (capturedErr) {
 						errorMessage += util.format(' Error output: \n %s', capturedErr);
 					}
-					future.throw(errorMessage);
+					future.throw(new Error(errorMessage));
```

Comment:
Because otherwise we throw string, which is allowed by the language but not a good practice.